### DISCUSSION TITLE
Detection of new updates from AeroInsta

### DISCUSSION THREAD

**Discussion by update-freak** [ImranR98/Obtainium#1433] [Q&A] [answered]
Hi,
is it possible to detect updates of the app Aeroinsta (https://aeroinsta.com/)?

**Comment by DwainZwerg** [comment]
I opened the page and to be honest I find it phishy. However, there doesn't seem to be any way for Obtainium to do anything. The page works by pressing the Java Script button for a redirect, then pressing the Java Script button for a redirect on the redirected page. The download is then presumably also solved with JavaScript. The page is also protected by Cloudflare.

**Reply by DwainZwerg** [reply, marked as answer]
@update-freak Yes, it is really fishy ([keweonDNS (a DNS provider who correctly disconnected me after a short time) #186](https://www.android-hilfe.de/forum/android-sicherheit-antivirus-firewalls-datenschutz-verschluesselung.910/keweondns-infos-fakten-und-was-ist-keweon-wirklich.1058609-page-10.html#post-13826327): It has something to do with Emotet (very dangerous Windows virus/malware)).
